Framework/DataHandling/test/LoadILLTest.h

```diff
@@ -0,0 +1,110 @@
+// Mantid Repository : https://github.com/mantidproject/mantid
+//
+// Copyright &copy; 2020 ISIS Rutherford Appleton Laboratory UKRI,
+//     NScD Oak Ridge National Laboratory, European Spallation Source
+//     & Institut Laue - Langevin
+// SPDX - License - Identifier: GPL - 3.0 +
+#ifndef MANTID_DATAHANDLING_LOADILLTEST_H_
+#define MANTID_DATAHANDLING_LOADILLTEST_H_
+
+#include <cxxtest/TestSuite.h>
+
+#include "MantidAPI/AnalysisDataService.h"
+#include "MantidAPI/MatrixWorkspace.h"
+#include "MantidDataHandling/Load.h"
+#include "MantidKernel/ConfigService.h"
+
+using Mantid::API::AnalysisDataService;
+using Mantid::API::MatrixWorkspace_const_sptr;
+using Mantid::DataHandling::Load;
+using Mantid::Kernel::ConfigService;
+
+class LoadILLTest : public CxxTest::TestSuite {
+public:
+  // This pair of boilerplate methods prevent the suite being created statically
+  // This means the constructor isn't called when running other tests
+  static LoadILLTest *createSuite() { return new LoadILLTest(); }
+  static void destroySuite(LoadILLTest *suite) { delete suite; }
+
+  void setUp() override {
+    ConfigService::Instance().appendDataSearchSubDir("ILL/D11/");
```

Comment:
Can you put the names of the instruments in an array, and then call append in a loop?